src/Task/TwigCsFixer.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Formatter\ProcessFormatterInterface;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Config\ConfigOptionsResolver;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use GrumPHP\Fixer\Provider\FixableProcessResultProvider;
+use Symfony\Component\Process\Process;
+
+/**
+ * @extends AbstractExternalTask<ProcessFormatterInterface>
+ */
+class TwigCsFixer extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): ConfigOptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'paths' => ['.'],
```

Comment:
Wouldn't it be better to provide an empty array here?
As far as I can tell from the tool, it will work that way as well.

Reply:
Done!